Dockerfile

```diff
@@ -1,7 +1,16 @@
-FROM golang:alpine3.11
+FROM golang:alpine3.11 as base
 
 WORKDIR /app
 COPY . /app
 
+ENV CGO_ENABLED=0
 RUN go build -o bin/naveed
-CMD ["/app/bin/naveed"]
+
+FROM busybox
+
+COPY --from=base /app/bin/naveed /usr/bin/naveed
+COPY --from=base /app/naveed.ini /
+
+EXPOSE 8465/tcp
```

Comment:
`8465` is just the default, so this assumes people didn't customize this setting in `naveed.ini` - which should generally be ok, but might result in confusion?

Reply:
yeah .. you can remove it as well. For docker images it would be more explicit when the port (and the listening address) could be passed as options or env vars